person.rb

```diff
@@ -22,6 +23,10 @@ def correct_name
     @name
   end
 
+  def add_rental(book, date)
+    @rentals.push(book, date, self)
+  end
```

Comment:
- [ ] With what we have in the rental class you need to refactor this method to instantiate a rental.

  ```
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
  ```